### ISSUE TITLE
Nextcloud: Request Entity Too Large

### ISSUE THREAD

**Issue by Steiniche** [Wonderfall/dockerfiles#160]
Hello,

Thanks for an awesome nextcloud container.
However, I current have a problem when uploading files which gives an error: Request Entity Too Large.
It seems to be triggered by files larger than 1MB.
From what I can search on google this can be caused by either Nginx, PHP, or Nextcloud, however they are all running with default settings from this repo which is 10GB.
Looking in the nextcloud admin section confirms that PHP Upload max size: 10.0 GB.
Looking inside the container the nginx.conf has the following set: client_max_body_size 10G;

How to reporduce:

Run the lastest 11.0 container (NEXTCLOUD_VERSION=11.0.2)  following the instructions from the hub.docker.com page.
Upload a file (https://upload.wikimedia.org/wikipedia/commons/9/97/BABOON-e.JPG) either via. the web UI or via. the desktop client.
This will produce an error saying "Request Entity Too Large"

Hope somebody can help figuring this one out!

**Comment by Steiniche** [comment]
I finally figured it out.
I have my nextcloud behind a jwilder/nginx-proxy and somehow it was working but after a server restart something must have changed.
The 413 error was sent from the nginx-proxy and not from the nginx within the wonderfall/nextcloud container.

The solution was to add a config file named exactly like the VIRTUAL_HOST in the vhost.d folder.
I used the default proxy config (see jwilder/nginx-proxy) and added client_max_body_size 10G;
More information can be found here: https://github.com/jwilder/nginx-proxy#per-virtual_host
